src/middleware/stat.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * # Middleware: StatCostTime
+ *
+ * For test only.
+ *
+ * ## Midlleware Scope
+ *
+ * For local.
+ *
+ * ## Midlleware Config
+ *
+ * No config.
+ *
+ *
+ * <br></br>
+ * Check [index](../modules/middleware.html) for more middleware.
+ *
+ * @module
+ */
+import Koa from 'koa';
+import { Provide } from '../util/container';
+import { Middleware } from '../util/web';
+
+@Provide()
+export class StatCostTime {
```

Comment:
What's the usage of this StatCostTime middleware?

Reply:
For test coverage, I wanted to check if there can be two or more `@Guard` to decorate one mehtod ([implement](https://github.com/Lellansin/test-backend/blob/master/src/util/web.ts#L145)) and I test it [here](https://github.com/Lellansin/test-backend/blob/master/src/controller/user.ts#L114)